http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/StaticKeyManagerFactory.java

```diff
@@ -24,11 +25,11 @@
  */
 @SdkInternalApi
 public final class StaticKeyManagerFactory extends KeyManagerFactory {
-    private StaticKeyManagerFactory(KeyManager[] keyManagers) {
+    private StaticKeyManagerFactory(List<KeyManager> keyManagers) {
```

Comment:
As discussed we're going to revert the List thing and keep it at Arrays because we don't want to touch the TlsKeyManagersProvider